### PR TITLE
sd/another test

### PR DESCRIPTION
**Pull request by davidmav19**

### CODE REVIEW

**Review by bnovikov09 — APPROVED** [review]
ok

File: src/test/java/school/redrover/DavidMTest.java [hunk 1]
```diff
@@ -65,5 +65,9 @@ public void TestSecond() throws InterruptedException {
       //test
```
Comment (EvgenyZagrebelnikov) [inline src/test/java/school/redrover/DavidMTest.java:65]: Delete comments

**Review by evanmaiqa — DISMISSED** [review]
system.out.println cannot be written in tests


just approve it

**Review by bnovikov09 — APPROVED** [review]
ok

**Review by ArtyomDulya — APPROVED** [review]

**Pull request merged**